### DISCUSSION TITLE
Ability to edit time zone layout?

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1269] [Q&A] [answered]
Hi!

Our intergroup would like to change how the time zone info is displayed on the individual meeting page because we have people joining our Zoom meetings from different areas and we want to more clearly spell out the time zone information.  For instance, it currently states PST.  We would like it to say US Pacific Time Zone.  Is there a way to do that?

**Comment by tim-rohrer** [comment, marked as answer]
Hi. For the purpose of clarity, the native TSML public meeting page does not display timezone information, and you are seeing this in TSML-UI. 

Regardless, we do not have a way to show full time zones in the manner you are asking.

C4R members are discussing ways that we can improve time zone support, especially for online meetings. 

For now, if you really want to spell it out, perhaps adding the timezones to the notes will work. However, I'm hopeful most people can Google or figure out PST
